app/lib/features/attachments/widgets/attachment_item.dart

```diff
@@ -43,49 +42,55 @@ class AttachmentItem extends ConsumerWidget {
       decoration: BoxDecoration(
         color: containerColor,
         borderRadius: BorderRadius.circular(8),
-        border: Border.all(color: borderColor),
+        border: Border.all(color: greyColor),
```

Comment:
I don't understand the rules for when we are adding new colors and when we move away from the theme.... I thought the goal was to move away from consts and use the theme, but now this _adds_ a theme-specific global again ...?!?

Reply:
Yes true, agree with you. Actually I was finding proper theme attribute for grey color but haven't figure it out. So I have added to with const and noted it on TO-DO list.

I would have to add FIXME comment for better note taking. Sorry for missing it.